web-ui/src/main/resources/catalog/views/default/templates/home.html

```diff
@@ -44,23 +44,20 @@ <h1 data-translate="">getstarted</h1>
 
             An icon could be provided as a CSS class name or
             a pattern matching a URL resolving to an image.
-
-            Properties
-            <div data-flfilter="categories"
-               data-title=""
-               data-icon-path="../../images/category/{{['@name']}}.png">
             -->
           <div class="row">
-            <div data-ng-repeat="(key, facet) in searchInfo.facet['topicCats']"
-              class="col-xs-6 col-sm-4 col-lg-3">
-              <a class="thumbnail gn-icon-bg" data-ng-href="#/search?topicCat={{facet['@name']}}">
-                <i class="fa fa-3x fa-table gn-icon gn-icon-{{facet['@name']}}"></i><br/>
-                <span class="gn-icon-label">{{facet['@label']}}</span>
+            <div data-ng-repeat="(key, facet) in searchInfo.facet['orgNames'] | limitTo:12"
```

Comment:
Got it.  Thanks for the clarification.

On Thu, Feb 26, 2015 at 5:35 PM, paul van genuchten <
notifications@github.com> wrote:

> In web-ui/src/main/resources/catalog/views/default/templates/home.html
> <https://github.com/geonetwork/core-geonetwork/pull/824#discussion_r25441380>
> :
>
> >              -->
> >            <div class="row">
> > -            <div data-ng-repeat="(key, facet) in searchInfo.facet['topicCats']"
> > -              class="col-xs-6 col-sm-4 col-lg-3">
> > -              <a class="thumbnail gn-icon-bg" data-ng-href="#/search?topicCat={{facet['@name']}}">
> > -                <i class="fa fa-3x fa-table gn-icon gn-icon-{{facet['@name']}}"></i><br/>
> > -                <span class="gn-icon-label">{{facet['@label']}}</span>
> > +            <div data-ng-repeat="(key, facet) in searchInfo.facet['orgNames'] | limitTo:12"
>
> yes they are jesse, but this configuration would indicate which of the
> facets is displayed as a teaser on the homepage, we now have topiccats and
> types, but other users might like inspire themes, protocols, organisations
> or ...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geonetwork/core-geonetwork/pull/824/files#r25441380>.
>